app/templates/releases/index.hbs

```diff
@@ -1,181 +1,100 @@
+
+{{release-timeline project=emberBetaProject}}
+
+<h2>Strategy</h2>
 <p>
-  LTS (Long Term Support) releases are done at least every four minor versions,
-  plus the last version before a new major release. An LTS release receives security
-  updates for 9 release cycles (54 weeks) and bugfixes for 5 release cycles
-  (30 weeks).
+  Ember is built by people who are at the front lines of building and upgrading their company's apps.
+  They want the latest features without the burn of breaking changes, and so Ember follows
+  a different release strategy than most other JavaScript tools.
 </p>
 
-<table class="lts-schedule table">
-  <thead>
-    <tr>
-      <th colspan="4">
-        LTS Support Schedule
-      </th>
-    </tr>
-    <tr>
-      <td>LTS Version</td>
-      <td>Released</td>
-      <td>Bugfixes until:</td>
-      <td>Security patches until:</td>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>
-        3.8
-      </td>
-      <td>
-        Feb 2019
-      </td>
-      <td>
-        Sep 2019
-      </td>
-      <td>
-        Feb 2020
-      </td>
-    </tr>
-    <tr>
-      <td>
-        3.4
-      </td>
-      <td>
-        Oct 2018
-      </td>
-      <td>
-        May 2019
-      </td>
-      <td>
-        Oct 2019
-      </td>
-    </tr>
-    <tr>
-      <td>
-        2.18
-      </td>
-      <td>
-        Feb 2018
-      </td>
-      <td>
-        Sept 2018
-      </td>
-      <td>
-        Feb 2019
-      </td>
-    </tr>
-  </tbody>
-</table>
+<h2>Our goals</h2>
+<ul>
+  <li>
+    Add new features in a way that doesn't break existing apps, through
+    backwards compatibility and optional feature flags
+  </li>
+  <li>
+    Maintain LTS (long term support) versions for 54 weeks, so that teams who upgrade their apps
+    infrequently can keep getting security updates and bugfixes
+  </li>
+  <li>
+    Make a minor release about every six weeks, so teams that use Ember can plan their work
+  </li>
+  <li>
+    Follow a public RFC (request for comments) process so that all users and companies
+    can participate in proposing and evaluating new features
+  </li>
+  <li>
+    Provide automated tooling for upgrades and syntax changes
+  </li>
+  <li>
+    Only cut a new major version (i.e. make a breaking change) when we really, really have to
+  </li>
+  <li>
+    Give developers a way to test drive the latest and greatest features, on their own terms.
+  </li>
+</ul>
 
+<h2>How Ember uses SemVer</h2>
 <p>
-  The Ember project adheres to
-  <a href="http://semver.org/">semantic versioning</a>. Releases are named
-  according to a <code>MAJOR.MINOR.PATCH</code> scheme, and only
-  <code>MAJOR</code> versions releases may change or remove public APIs after
-  deprecation. <code>MINOR</code> versions may introduce new features so long
-  as they are backwards compatible, and <code>PATCH</code> releases may include
-  bug or security fixes.
+  SemVer stands for
+  <a href="http://semver.org/">Semantic Versioning</a>,
```

Comment:
```suggestion
  <a href="http://semver.org/" target="_blank" rel="noopener">Semantic Versioning</a>,
```

Reply:
@MelSumner mentioned that we're not doing target blank for accessibility reasons :+1

Reply:
Where can I read about this?